### PR TITLE
Refactor Build stages

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,9 +10,9 @@ python:
 os:
     - linux
 stages:
-  - lint
   - test
   - integration
+  - lint
```
Comment (thedrow) [inline .travis.yml]: No.
Lint should be triggered first.
There's no point to run the entire build if the code is not up to our standards.

Comment (auvipy) [reply .travis.yml]: it is un needed to check the code standard before the code passes the test. if you disagree lets open an issue and the community vote. its annoying to see the whole build stopped just for a simple white space or comma. I even saw django channels following this. Lets open an issue regarding test stage ordering than

Comment (thedrow) [reply .travis.yml]: Ok let's open an issue...

Comment (auvipy) [reply .travis.yml]: issue is open

**Pull request closed without merging**